README.md

```diff
@@ -8,22 +8,77 @@
   <br>
 </h1>
 
-# Features
-- hot reloading
-- basic functionality
+This module implements the [`rdk:component:sensor` API](https://docs.viam.com/components/sensor) and provides two sensor models:
 
-# Models
 ```
 zaporter:sds011:v1
 zaporter:sds011:v1-fake
 ```
-# Example Config
+
+The `zaporter:sds011:v1` model supports the SDS011 Nova PM air quality sensor.
+The `zaporter:sds011:v1-fake` can be used for testing the module without hardware.
+
+[!NOTE]
+> For more information, see [Modular Resources](https://docs.viam.com/registry/#modular-resources).
+
+## Features
+
+- Hot reloading
+- Basic functionality
+
+## Configure your SDS011 sensor
```

Comment:
```suggestion
## Configure your `sds011:v1` sensor
```
Have been doing this in template to make it slightly more clear

Reply:
Ahh I thought about it but figured I can smush fake and real into one heading if I don't do monospace. Does that work?

Reply:
I like just `SDS011`